test/e2e/e2e_suite_test.go

```diff
@@ -9,18 +9,35 @@ import (
 )
 
 var (
-	veleroCLI        string
-	kibishiNamespace string
-	cloudPlatform    string // aws, vsphere, azure
+	veleroCLI            string
+	veleroImage          string
+	cloudCredentialsFile string
+	bslConfig            string
+	bslBucket            string
+	bslPrefix            string
+	vslConfig            string
+	pluginProvider       string
```

Comment:
These could be 1 or 2 lines by using commas.

Reply:
Fair enough.